api/provider/discord.go

```diff
@@ -76,11 +77,15 @@ func (g discordProvider) GetUserData(ctx context.Context, tok *oauth2.Token) (*U
 
 	var avatarURL string
 	extension := "png"
-	// https://discord.com/developers/docs/reference#image-formatting-cdn-endpoints:
-	// In the case of the Default User Avatar endpoint, the value for
-	// user_discriminator in the path should be the user's discriminator modulo 5
 	if u.Avatar == "" {
-		avatarURL = fmt.Sprintf("https://cdn.discordapp.com/embed/avatars/%d.%s", u.Discriminator%5, extension)
+		if intDiscriminator, err := strconv.Atoi(u.Discriminator); err != nil {
```

Comment:
will the discriminator returned from discord be "#0999" or "0999" here? if it's the former, you'll need to strip out the "#" before passing it into `strconv.Atoi` or it will always return an error

Reply:
maybe @PhaxeNor would know? :)

Reply:
Discord only returns numbers (as string ofc), so "0999" in this case.